templates/user/verify.html

```diff
@@ -1,9 +1,141 @@
+            color: inherit !important;
+            text-decoration: none !important;
+            font-size: inherit !important;
+            font-family: inherit !important;
+            font-weight: inherit !important;
+            line-height: inherit !important;
+        }
+
+        /* MOBILE STYLES */
+        @media screen and (max-width:600px) {
+            h1 {
+                font-size: 32px !important;
+                line-height: 32px !important;
+            }
+        }
+
+        /* ANDROID CENTER FIX */
+        div[style*="margin: 16px 0;"] {
+            margin: 0 !important;
+        }
+    </style>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Roboto:ital,wght@1,500&display=swap" rel="stylesheet"> 
+
+    <style>
+        body {
+          font-family: 'Roboto', sans-serif;
+        }
+      </style>
+</head>
+
+<body style="background-color: #f4f4f4; margin: 0 !important; padding: 0 !important;">
+    <!-- HIDDEN PREHEADER TEXT -->
+    <div style="display: none; font-size: 1px; color: #fefefe; line-height: 1px;max-height: 0px; max-width: 0px; opacity: 0; overflow: hidden;"> We're thrilled to have you here! Get ready to dive into your new account. </div>
+    <table border="0" cellpadding="0" cellspacing="0" width="100%">
+        <tr>
+            <td bgcolor="#f4f4f4" align="center" style="padding: 0px 10px 0px 10px;">
+                <table border="0" cellpadding="0" cellspacing="0" width="100%" style="max-width: 600px;">
+                    <tr>
+                        <td bgcolor="#ffffff" align="center" valign="top" style="padding: 5px 15px 30px 15px; border-radius: 4px 4px 0px 0px; color: #111111; font-family: 'Roboto', Helvetica, Arial, sans-serif; font-size: 48px; font-weight: 400; letter-spacing: 4px; line-height: 48px;">
+                             <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRzkuTpE0q2DJnKC3XJMbmMUA_WZRYcBlefOHSmgMp71IukHAgx&usqp=CAU" width="150" height="100" style="display: block; border: 0px;" />
+                             <h1 style="font-size: 48px; font-weight: 400; margin: 2;">{{ name }}!</h1>
+                        </td>
+                    </tr>
+                </table>
+            </td>
+        </tr>
+        <tr>
+            <td bgcolor="#f4f4f4" align="center" style="padding: 0px 10px 0px 10px;">
+                <table border="0" cellpadding="0" cellspacing="0" width="100%" style="max-width: 600px;">
+                    <tr>
+                    
+                        <td bgcolor="#ffffff" align="left" style="padding: 5px 15px 30px 15px; color: #666666; font-family: 'Roboto', Helvetica, Arial, sans-serif; font-size: 18px; font-weight: 400; line-height: 25px;">
+                            <p style="margin: 0;"> We're thrilled to have you become apart of Crane Cloud! Get ready to dive into your new account. Let's help you get started. First, you need to confirm your account. Just press the button below.</p>
```

Comment:
not **apart** guess you meant **a part**

Reply:
Yeah, I second this. @MutegekiHenry Please take note and make that correction.

Reply:
@neelxie thanks let me change that

Reply:
Done